src/app/app-routing.module.ts

```diff
@@ -2,14 +2,20 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
+// Not Found Component
+import {NotFoundComponent} from './not-found/not-found.component';
+
 /**
  * Fallback to this route when no prior route is matched.
  */
 const routes: Routes = [
+  {
+    path: '404',
+    component: NotFoundComponent
+  },
   {
     path: '**',
-    redirectTo: '',
-    pathMatch: 'full'
+    redirectTo: '/404'
```

Comment:
As I told you earlier, remove the redirect route and just simply add a new path for 404. Update the code with this:
```typescript
{
    path: '**',
    component: PageNotFoundComponent
}
```

Reply:
Sorry I believe I didn't reapply the changes when I squashed my commits to the wrong version.